### PR TITLE
dist: Recommend more efficient encoder

### PR DESCRIPTION
**Pull request by okurz**

### CODE REVIEW

**Review by perlpunk — APPROVED** [review]

**Review by Martchus — CHANGES_REQUESTED** [review]

File: dist/rpm/os-autoinst.spec [hunk 1]
```diff
@@ -110,6 +110,8 @@ Recommends:     perl(Inline::Python)
 Recommends:     python3-Pillow-tk
 # Optional dependency for QEMU's built-in samba service (enabled via QEMU_ENABLE_SMBD=1)
 Recommends:     samba
+# More efficient video encoding is done automatically if ffmpeg is present
+Recommends:     ffmpeg-4
```
[the remark this answers is not in the document]
Comment (okurz) [reply dist/rpm/os-autoinst.spec:114]: err, right. Fun with this versioned package names :) I looked into https://build.opensuse.org/package/view_file/openSUSE:Factory/ffmpeg-6/ffmpeg-6.spec?expand=1 but could not find any sensible "Provides" that we could rely upon. Anybody has a good idea?

[the remark this answers is not in the document]
Comment (Martchus) [reply dist/rpm/os-autoinst.spec:114]:
```suggestion
Recommends:     ffmpeg
```

[the remark this answers is not in the document]
Comment (okurz) [reply dist/rpm/os-autoinst.spec:114]: No, wrong operating system:

```
$ podman run --rm -it registry.opensuse.org/opensuse/leap:15.5
cb0c41737f7e:/ # zypper se --details ffmpeg
…

S | Name                         | Type    | Version                  | Arch   | Repository
--+------------------------------+---------+--------------------------+--------+----------------
  | cmus-plugin-ffmpeg           | package | 2.10.0-bp155.1.10        | x86_64 | Main Repository
  | ffmpeg                       | package | 3.4.2-150200.11.28.1     | x86_64 | Main Repository
  | ffmpeg-4                     | package | 4.4-150400.3.15.1        | x86_64 | Main Repository
```

[the remark this answers is not in the document]
Comment (mimi1vx) [reply dist/rpm/os-autoinst.spec:114]: more like something

[the remark this answers is not in the document]
Comment (Martchus) [reply dist/rpm/os-autoinst.spec:114]: Although we probably even wanted ffmpeg-6 or whatever is currently the latest.

[the remark this answers is not in the document]
Comment (okurz) [reply dist/rpm/os-autoinst.spec]: `Recommends:            ffmpeg >= 4` looks good, done

**Review by kalikiana — APPROVED** [review]

**Comment by okurz** [comment]
@Martchus please re-evaluate

**Review by Martchus — APPROVED** [review]
I've just tested this in a Tumbleweed and Leap 15.5 container and it looks like this version constraint would really install the latest version each system provides (ffmpeg-6 or ffmpeg-4).

**Pull request merged**